staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -172,3 +175,8 @@ func (f *FieldManager) toVersioned(obj runtime.Object) (runtime.Object, error) {
 func (f *FieldManager) toUnversioned(obj runtime.Object) (runtime.Object, error) {
 	return f.objectConverter.ConvertToVersion(obj, f.hubVersion)
 }
+
+func (f *FieldManager) buildManagerInfo(prefix, version string) string {
+	timestamp := time.Now().Format("20060102-150405")
+	return fmt.Sprintf("%s-%s@%s", prefix, version, timestamp)
```

Comment:
I suspect we have to use the timezone in there, because we might have multiple apiserver in different area, and then you don't know what the timestamp actually means (or always make it UTC or something).

Reply:
Is there a styleguide or something on timestamp values inside k8s?
Adding the timezone to the end would be trivial, but if there are rules i would like to have it consistent.